static/js/modules/tables.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
+
+/* global require, module, API_LOCATION, API_VERSION, API_KEY */
+
+var $ = require('jquery');
+var _ = require('underscore');
+var URI = require('URIjs');
+require('datatables');
+
+function yearRange(first, last) {
+  if (first === last) {
+    return first;
+  } else {
+    return first.toString() + ' - ' + last.toString();
+  }
+}
+
+function mapFilters(filters) {
+  return _.reduce(filters, function(acc, val) {
+    if (val.value) {
+      if (acc[val.name]) {
+        acc[val.name].push(val.value);
+      } else {
+        acc[val.name] = [val.value];
+      }
+    }
+    return acc;
+  }, {});
+}
+
+var parsedFilters;
+
+function buildCycle(datum) {
+  if (parsedFilters.cycle) {
+    var cycles = _.intersection(
+      _.map(parsedFilters.cycle, function(cycle) {return parseInt(cycle);}),
+      datum.cycles
+    );
+    return '?cycle=' + _.max(cycles);
+  } else {
+    return '';
+  }
+}
+
+var candidateColumns = [
+  {
+    data: 'name',
+    render: function(data, type, row, meta) {
+      var anchor = $('<a>');
```

Comment:
Sure, I'll do a pass through the patch and make sure jquery elements are prefixed with `$`.